packages/client/src/LensClient.spec.ts

```diff
@@ -6,34 +6,30 @@ const testConfig = {
 };
 
 describe(`Given the LensClient configured for sandbox`, () => {
-  // BE CAREFUL, below test causes race condition
-  // with setupTestCredentials helper used in other tests
-  // the same wallet is authenticated twice
+  const client = new LensClient(testConfig);
 
-  // describe(`when authenticating with wallet signature`, () => {
-  //   it(`should authenticate without throwing`, async () => {
-  //     const wallet = setupTestWallet();
-  //     const client = LensClient.init(testConfig);
-  //     const address = await wallet.getAddress();
-  //     const challenge = await client.generateChallenge(address);
-  //     const signature = await wallet.signMessage(challenge);
+  describe(`when checking authentication features exposed by the client`, () => {
+    it(`should offer ${LensClient.prototype.generateChallenge.name} function`, async () => {
+      expect(typeof client.generateChallenge).toEqual('function');
+    });
 
-  //     await expect(client.authenticate(address, signature)).resolves.not.toThrow();
-  //   });
-  // });
+    it(`should offer ${LensClient.prototype.authenticate.name} function`, async () => {
+      expect(typeof client.authenticate).toEqual('function');
+    });
+
+    it(`should offer ${LensClient.prototype.isAuthenticated.name} function`, async () => {
```

Comment:
LensClient.spec is only testing the interface exposed to end-user

Reply:
I don't know how much value this type of tests bring... but in general this class of tests could also be done with TS typedef tests. It would test not just the presence of functions but also their arguments and return type.

I am not suggesting to action this in this PR.

Reply:
they will be removed in https://avara.height.app/lens-sdk/T-6208